### DISCUSSION TITLE
Historical price data

### DISCUSSION THREAD

**Discussion by ghost** [fja05680/sp500#5] [Q&A] [answered]
Hi,

i want to purchase historical end of day price data to backtest the momentum strategy in clenows book. Could you tell me, which data package i should buy on eoddata.

And which "Data Format Options" should i choose for zipline.

Thank you very much in advance. 

Nice greetings from germany

**Comment by fja05680** [comment, marked as answer]
Hi Psydra,

It appears you need the "US Equity Historical data Bundle" for $200 from eoddata.  That should contain all the S&P 500 equitiesfor 20 years.  You might email them to be sure.  As to zipline, I have only a little experience with that software.  You should be able to google search or ask in a zipline forum for that answer.  Best of Luck,

Farrell

**Comment by Hvm123** [comment]
Historical options and futures data available from 2010-2022 JULY.
One min ticks in the below mentioned format.
Open interest + Volume are available too with the instrument.
DM IF NEEDED FOR BACKTESTING NEEDS. 
whatsap 8584079835
![Uploading 3C01FEE6-284C-42BD-9D7F-25EB356155A5.jpeg…]()
